[2023-04-08T10:16:53.171+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T18:20:00+00:00 [queued]>
[2023-04-08T10:16:53.187+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T18:20:00+00:00 [queued]>
[2023-04-08T10:16:53.190+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:53.192+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-08T10:16:53.194+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:53.213+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-06 18:20:00+00:00
[2023-04-08T10:16:53.221+0000] {standard_task_runner.py:55} INFO - Started process 13717 to run task
[2023-04-08T10:16:53.226+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-06T18:20:00+00:00', '--job-id', '322', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmp288s6st9']
[2023-04-08T10:16:53.231+0000] {standard_task_runner.py:83} INFO - Job 322: Subtask respond
[2023-04-08T10:16:53.353+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-06T18:20:00+00:00 [running]> on host cfece00ff772
[2023-04-08T10:16:53.486+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T18:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-06T18:20:00+00:00
[2023-04-08T10:16:53.490+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-08T10:16:53.538+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230406T182000, start_date=20230408T101653, end_date=20230408T101653
[2023-04-08T10:16:53.600+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-08T10:16:53.649+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
